modules/storages/app/common/storages/peripherals/nextcloud_registry.rb

```diff
@@ -37,7 +37,7 @@ module Peripherals
         register(:file_ids, StorageInteraction::Nextcloud::FileIdsQuery)
         register(:file_info, StorageInteraction::Nextcloud::FileInfoQuery)
         register(:files_info, StorageInteraction::Nextcloud::FilesInfoQuery)
-        register(:files, StorageInteraction::Nextcloud::FilesQuery)
+        register(:files, Adapters::Nextcloud::Queries::Files)
```

Comment:
`Registries` should be moved to their adapter folders (maybe we could add a way to auto-register? 🤔)